run.sh

```diff
@@ -15,6 +15,8 @@ if [[ $? -eq 0 ]]; then
 
     str_num=$(echo $status | sed -nr "s/.*на странице №([[:digit:]]{1,4})!.*/\1/p")
     echo "Задача $zad найдена на странице №$str_num!"
+    
+    ./open.sh $sem $str_num $zad
```

Comment:
I suppose it would be better to add options to a single script file instead of creating multiple different scripts. This, in prospect, will allow for easier installation.

Reply:
I'm not trying to say that these scripts work perfect and support text streams, but I tryed to use Unix-way concept while working on them. Maybe you're right, because of the original script size (I can agree that it can be too small to apply this concept).
So, I can unite these scripts into one file (please specify whether you want to see one shared file or a separate `open.sh` file with all scripts connected to Koryavnik downloading&opening) or create Makefile with an installation target (as my main thought about, this target will copy all `gnu-koryavov` scripts to `/usr/local/bin` and create `KORYAVNIKS/` dir into the user's home directory).

Reply:
You are right: unix way is the best way, but in this case all functionality correspond to single  program instance. Also, your installation idea lgtm, but stick to managing only one script file. Makefike here can be replaced in thjs  ase eith separate `install.sh` script. Also, .config file should be put intto users home/ dir.